contribute/creating_package.md

```diff
@@ -16,7 +16,7 @@ In case of any issues, remember that [help](https://github.com/elizavetasemenova
 
 ## Starting a package
 
-1. You need to create a folder to store a new package. For example, if you use Atom as an IDE, create a new folder called `tiddlywinks.jl`. Then, open the GitHub panel and select **Initialize and publish on GitHub**. Now, you can go to your GitHub page and see that 'tiddlywinks.jl' has appeared there, as well as locally.
+1. You need to create a folder to store a new package. For example, if you use Atom as an IDE, create a new folder called `Tidlywinks.jl` (packages in Julia generally have a capital first letter). Then, open the GitHub panel and select **Initialize and publish on GitHub**. Now, you can go to your GitHub page and see that 'Tidlywinks.jl' has appeared there, as well as locally.
```

Comment:
I think you accidentally dropped a `d` in the name here.

Reply:
yep, corrected